src/devices/src/virtio/net/event_handler.rs

```diff
@@ -26,7 +27,10 @@ impl EventHandler for Net {
             _ if source == virtq_tx_ev_fd => self.process_tx_queue_event(),
             _ if source == rx_rate_limiter_fd => self.process_rx_rate_limiter_event(),
             _ if source == tx_rate_limiter_fd => self.process_tx_rate_limiter_event(),
-            _ => error!("Unknown event. No handling was done."),
+            _ => {
+                error!("Unknown event. No handling was done.");
```

Comment:
This message should be "unknown event source" or similar. The event is known, the source is the one that it's not.